tests/flytekit/unit/models/test_tasks.py

```diff
@@ -182,10 +182,14 @@ def test_sidecar_task():
     pod_spec = generated_pb2.PodSpec()
     container = generated_pb2.Container(name="containery")
     pod_spec.containers.extend([container])
-    obj = task.SidecarJob(pod_spec=pod_spec, primary_container_name="primary")
+    obj = task.SidecarJob(pod_spec=pod_spec, primary_container_name="primary",
+                          annotations={'a1': 'a1'}, labels={'b1': 'b1'},
```

Comment:
can we add one more test for when annotations/labels is not provided - and test that the `task.SidecarJob.from_flyte_idl(obj.to_flyte_idl())` pattern still works.

Reply:
Added